frontend-public/src/styles/settings/theme.scss

```diff
@@ -0,0 +1,8 @@
+// theme.scss holds overrides for ant-design
+// refer to (https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less) for ant design variable names;
+$blue: #003366;
+$white: #f1f1f2;
+$violet: #5e46a1;
```

Comment:
I think all of these can be removed actually, they aren't being used anywhere. This file can be left blank unless you're overriding anything with those variables.

Reply:
made the file empty, I needed it for the web pack loader but forgot about the variables in it. Thanks.